power/nut-client/pkg.yaml

```diff
@@ -0,0 +1,101 @@
+name: nut-client
+variant: scratch
+shell: /toolchain/bin/bash
+dependencies:
+  - stage: base
+  - image: "{{ .PKGS_PREFIX }}/openssl:{{ .PKGS_VERSION }}"
+  - image: "{{ .PKGS_PREFIX }}/util-linux:{{ .PKGS_VERSION }}"
+steps:
+  - sources:
+      - url: https://networkupstools.org/source/2.8/nut-2.8.0.tar.gz
```

Comment:
let's move the url and sha's to `vars.yaml` under `power` folder, and add a `renovate` comment. Eg: https://github.com/siderolabs/extensions/blob/main/nvidia-gpu/vars.yaml#L1

Reply:
updated to match nvidia-gpu/vars.yaml